src/platform/windows/PolicyConfig.h

```diff
@@ -10,7 +10,7 @@
 
 #include <mmdeviceapi.h>
 
-#ifdef __MINGW32__
+#ifdef _WIN32
```

Comment:
This is in `platform/windows` so `_WIN32` should always be defined when building it.